SUMO-239629: Updated aws api gateway app (#4029)
SumoLogic/sumologic-documentation: docs/integrations/amazon-aws/api-gateway.md

   10. Click **Save**.
   11. Save the given URL of the source for next step.
2. [Create Stack](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-logs-source/#cloudformation-template) in AWS console with given CloudFormation Template.

> 3. Create a log group in CloudWatch Logs by referring to [AWS documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Working-with-log-groups-and-streams.html). (Make sure to have log group name convention as `/aws/apigateway/<apiid>/<stagename>`)

Reviewer: ```suggestion
3. Create a log group in CloudWatch Logs by referring to the [AWS documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Working-with-log-groups-and-streams.html). Make sure to set your log group name convention as `/aws/apigateway/<apiid>/<stagename>`.
```

4. Follow below steps for enabling access logs for each respective API type:
   :::note
   Make sure to remove remove :* from the end while adding Access log destination ARN.
   :::

>    * Enable Access logs for REST APIs by referring to [AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html) and  when you specify the `Log format` field use the below JSON.

Reviewer 2: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html#set-up-access-logging-using-console lets use this link

> 3. Create a log group in CloudWatch Logs by referring to [AWS documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Working-with-log-groups-and-streams.html). (Make sure to have log group name convention as `/aws/apigateway/<apiid>/<stagename>`)
> 4. Follow below steps for enabling access logs for each respective API type:
>    :::note
>    Make sure to remove remove :* from the end while adding Access log destination ARN.
>    :::
>    * Enable Access logs for REST APIs by referring to [AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html) and  when you specify the `Log format` field use the below JSON.

Reviewer: ```suggestion
3. Create a log group in CloudWatch Logs by referring to the [AWS documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/Working-with-log-groups-and-streams.html). Make sure to have your log group name convention as `/aws/apigateway/<apiid>/<stagename>`.
4. Enable Access logs for each respective REST API type by referring to the [AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html#set-up-access-logging-using-console). When you specify the `Log format` field, use the below JSON.
   :::note
   Make sure to remove remove `:*` from the end while adding the Access log destination ARN.
   :::
```

Author: Here under the 4th point, we have written steps for 3 different API types(REST API, HTTP API, and WebSocket API). So we cannot make REST API's steps direct as 4th point.

Reviewer: ah, I missed those other steps! fixed

> 4. Follow below steps for enabling access logs for each respective API type:
>    :::note
>    Make sure to remove remove :* from the end while adding Access log destination ARN.
>    :::
>    * Enable Access logs for REST APIs by referring to [AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html#set-up-access-logging-using-console) and  when you specify the `Log format` field use the below JSON.

Reviewer: ```suggestion
4. Follow the below steps to enable access logs for each respective API type:
   :::note
   Make sure to remove remove `:*` from the end while adding Access log destination ARN.
   :::
   * Enable Access logs for REST APIs by referring to the [AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html#set-up-access-logging-using-console). When you specify the `Log format` field, use the below JSON.
```
